api/client/src/main/java/org/projectnessie/client/auth/oauth2/OAuth2Client.java

```diff
@@ -126,11 +128,12 @@ Tokens getCurrentTokens() {
     } catch (ExecutionException e) {
       Throwable cause = e.getCause();
       if (cause instanceof RuntimeException) {
+        cause.fillInStackTrace();
```

Comment:
How about throwing a new `RuntimeException` but adding `cause` as a "suppressed" exception? This way we preserve both stack traces for analysis.

Reply:
The original stack trace is really useless. It can only originate from `checkErrorResponse` below.

Reply:
Perhaps... but the call to re-fill the cause's stack trace unnerves me :) What do we gain by reusing the exception with a different stack trace vs. deriving a fresh exception from it?

Reply:
Just less clutter in the console. I'm fine removing this change.

Reply:
Thanks! I happen to appreciate the finer stack details when I have to dig into errors :)